scripts/setup.sh

```diff
@@ -43,6 +43,11 @@ function maybeInstallDependencies {
       shift
       ;;
 
+      --suite=*)
```

Comment:
Hm, my question still stands: why doesn't SHARDING etc. lead to the same error as well?